app/views/users/index.html.erb

```diff
@@ -1 +1,15 @@
 <h1>Users#index</h1>
+
+<ul>
+  <% @users.each do |user| %>
+    <li>
+      <div>
+        <img src=<%= user.photo %> alt="user photo">
+        <div> 
+          <h2><%= user.name %> #<%= user.id %></h2>
+          <p>Number of posts: <%= user.posts_counter %></p>
+        </div>
+      </div>
+    </li>
+  <% end %>
+</ul>
```

Comment:
Clicking on a User is suppose to take you to the user show page, In your case its not working , Kindly fix this issue 👍